src/mantine-core/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -497,10 +497,11 @@ export const MultiSelect = forwardRef<HTMLInputElement, MultiSelectProps>(
               rightSectionWidth,
               styles,
               size,
-              shouldClear: !disabled && clearable && _value.length > 0,
```

Comment:
This caused the bug where the clear button would not show if the input was also disabled.
![image](https://user-images.githubusercontent.com/9464690/155848821-85b94a7f-6028-4cac-8c07-6055e21f731c.png)